scripts/big_bg_status.js

```diff
@@ -0,0 +1,166 @@
+    display.oled.fillRect(127-batt, 58, batt, 5, 1, false); //fill battery gauge
+}
+
+//calculate timeago for BG
+if(bg && profile) {
+    var startDate = new Date(bg[0].date);
+    var endDate = new Date();
+    var minutes = Math.round(( (endDate.getTime() - startDate.getTime()) / 1000) / 60);
+    if (bg[0].delta) {
+        var delta = Math.round(bg[0].delta);
+    } else if (bg[1] && bg[0].date - bg[1].date > 200000 ) {
+        var delta = Math.round(bg[0].glucose - bg[1].glucose);
+    } else if (bg[2] && bg[0].date - bg[2].date > 200000 ) {
+        var delta = Math.round(bg[0].glucose - bg[2].glucose);
+    } else if (bg[3] && bg[0].date - bg[3].date > 200000 ) {
+        var delta = Math.round(bg[0].glucose - bg[3].glucose);
+    } else {
+        var delta = 0;
+    }
+
+    //display BG number, add plus sign if delta is positive 
+    display.oled.setCursor(0,0);
+    if (delta >= 0) {
+        display.oled.writeString(font, 3, ""+convert_bg(bg[0].glucose, profile), 1, false, 0, false);
+        display.oled.writeString(font, 1, "+"+stripLeadingZero(convert_bg(delta, profile)), 1, false, 0, false);
+        display.oled.writeString(font, 2, "  "+minutes+"m", 1, false, 0, false);
+    } else {
+        display.oled.writeString(font, 3, ""+convert_bg(bg[0].glucose, profile), 1, false, 0, false);
+        display.oled.writeString(font, 1, ""+stripLeadingZero(convert_bg(delta, profile)), 1, false, 0, false);
+        display.oled.writeString(font, 2, "  "+minutes+"m", 1, false, 0, false);
+    }
+}
+
+//calculate timeago for last successful loop
+if(stats) {
+    var date = new Date(stats.mtime);
+    var hour = date.getHours();
+    hour = (hour < 10 ? "0" : "") + hour;
+    var min  = date.getMinutes();
+    min = (min < 10 ? "0" : "") + min;
+
+    //display last loop time
+    display.oled.setCursor(0,57);
+    display.oled.writeString(font, 1, "Last loop at: "+hour+":"+min, 1, false, 0, false);
+}
+
+//parse and render COB/IOB
+if(iob && cob) {
+    display.oled.setCursor(0,23);
+    display.oled.writeString(font, 1, "IOB:", 1, false, 0, false);
+    display.oled.writeString(font, 2, " "+iob[0].iob+'U', 1, false, 0, false);
+    display.oled.setCursor(0,39);
+    display.oled.writeString(font, 1, "COB:", 1, false, 0, false);
+    display.oled.writeString(font, 2, " "+cob.mealCOB+'g', 1, false, 0, false);
+}
+
+display.oled.dimDisplay(true); //dim the display
+display.oled.update(); // write buffer to the screen
+  
+fs.readFile(openapsDir+"/preferences.json", function (err, data) {
+  if (err) throw err;
+  preferences = JSON.parse(data);
+  if (preferences.wearOLEDevenly.includes("off")) {
+    display.oled.invertDisplay(false);
+  }
+  else if (preferences.wearOLEDevenly.includes("nightandday") && (clockHour >= 20 || clockHour <= 8)) {
+    display.oled.invertDisplay(false);
+  }
+  else if (preferences.wearOLEDevenly.includes("nightandday") && (clockHour <= 20 || clockHour >= 8)) {
```

Comment:
I think this will work, but only because it's after the night-time check.  It probably should be `clockHour <= 20 && clockHour >= 8`, not `||`.